{"name": "test_when_there_is_no_video_playout_a_waiting_video[chromium]", "status": "broken", "statusDetails": {"message": "requests.exceptions.ConnectionError: HTTPConnectionPool(host='10.189.200.6', port=80): Max retries exceeded with url: /api/v1/authenticate (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11678f9d0>: Failed to establish a new connection: [Errno 51] Network is unreachable'))", "trace": "E   OSError: [Errno 51] Network is unreachable\n\nThe above exception was the direct cause of the following exception:\nE   urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x11678f9d0>: Failed to establish a new connection: [Errno 51] Network is unreachable\n\nThe above exception was the direct cause of the following exception:\nE   urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='10.189.200.6', port=80): Max retries exceeded with url: /api/v1/authenticate (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11678f9d0>: Failed to establish a new connection: [Errno 51] Network is unreachable'))\n\nDuring handling of the above exception, another exception occurred:\nE   requests.exceptions.ConnectionError: HTTPConnectionPool(host='10.189.200.6', port=80): Max retries exceeded with url: /api/v1/authenticate (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x11678f9d0>: Failed to establish a new connection: [Errno 51] Network is unreachable'))"}, "parameters": [{"name": "browser_name", "value": "'chromium'"}], "start": 1758895245241, "stop": 1758895245241, "uuid": "47dd342e-0f56-4795-b357-475127d158d9", "historyId": "2abd14c0f6b30f6060a16d10dadb2089", "testCaseId": "fecfde811c7c2574f221e1dbe8bcc8ec", "fullName": "tests.Automation_tests.test_admin_playout#test_when_there_is_no_video_playout_a_waiting_video", "labels": [{"name": "feature", "value": "AdminPlayout"}, {"name": "parentSuite", "value": "tests.Automation_tests"}, {"name": "suite", "value": "test_admin_playout"}, {"name": "host", "value": "TR-GKJF6LPM4P"}, {"name": "thread", "value": "77029-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.Automation_tests.test_admin_playout"}]}